### ISSUE TITLE
[eslint-plugin-putout]: Error while loading rule `putout/putout` with class constructor type

### ISSUE THREAD

**Issue by ElPrudi** [coderaiser/putout#149]
I use this plugin with typescript and I use this type: `ctor: new (...args: any[]) => T` inside a function to describe an arbitrary constructor.

With this type inside my project, I get this error:

```

Oops! Something went wrong! :(

ESLint: 8.45.0

Error: Error while loading rule 'putout/putout': Node type 'TSConstructorType' is not supported yet: 'new (...args: any[]) => T'
```

**Comment by coderaiser** [comment]
Fixed in [`@putout/printer@2.85`](https://github.com/putoutjs/printer/releases/tag/v2.85.0) 🎉 . Please re-install 🐊.

Is it works for you?

**Comment by ElPrudi** [comment]
Yes, the error doesn't appear anymore. Thank you very much.
